pyproject.toml

```diff
@@ -4,7 +4,10 @@ build-backend = "poetry.core.masonry.api"
 
 
 [tool.nitpick]
-style = "https://raw.githubusercontent.com/wemake-services/wemake-python-styleguide/master/styles/nitpick-style-wemake.toml"
+style = [
+  "https://raw.githubusercontent.com/wemake-services/wemake-python-styleguide/master/styles/nitpick-style-wemake.toml",
+  "https://raw.githubusercontent.com/moowiz/workaround-nitpick-issue-418/main/black-flake8-compat-style.toml",
```

Comment:
This is nice, I like that. Could probably put it under this repo?

Reply:
I work on windows, and there's a bug in nitpick where it doesn't work correctly as a local file on windows. https://github.com/andreoliwa/nitpick/issues/418

Reply:
We could commit it and then point to the github main of this repo, but I don't know if that gives us much benefit.